frontend/src/components/Review/StarRate/StarRate.tsx

```diff
@@ -2,7 +2,7 @@ import { Button, Heading, Spacing, theme } from '@fun-eat/design-system';
 import styled from 'styled-components';
 
 import { SvgIcon } from '@/components/Common';
-import useStarRating from '@/hooks/useStarRate';
+import useStarRating from '@/hooks/review/useStarRating';
```

Comment:
```suggestion
import useStarRating from '@/hooks/review';
```

인덱스에 추가되었는데 인덱스에서 안 가져오고 있네용